routes/tasks.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express')
+const router = express.Router()
+const tasksController = require('../controllers/tasksController')
+
+/**
+ * @swagger
+ * /tasks/:
```

Comment:
I think we don't need an extra `/` here. Can you please changes it to `/tasks` ?